en/asset/audio.md

```diff
@@ -12,8 +12,18 @@ Currently, the engine's audio system can support the following formats:
   - `.mp4`
   - `.m4a`
 
-## Use of sound assets
+## loading mode of audio resources on the Web platform
 
-After adding an `AudioSource` to a `Node`, drag the imported audio asset from the __Asset Manager__ to the `Clip` of the node `AudioSource` to control the sound asset:
+Audio resources on the Web platform are special because the Web standard supports loading audio resources in two different ways as follows:
+- Web Audio: providing a relatively more modern interface for audio control, the audio resources will be cached in an audio buffer of the engine. The advantage of this approach is good compatibility and robust.
+- DOM Audio: by generating a standard element to play the audio resources, the cache is the audio element. When using standard audio elements to play audio resources, you may encounter some restrictions on some browsers. For example, browsers on iOS do not support setting volume, and all volume-related properties will not be available.
 
-![](audio/audiocilp.gif)
+The engine currently tries to load audio resources as Web Audio by default. If it detects that the browser does not support loading Web Audio, it will fall back to the DOM Audio mode.

```

Comment:
```suggestion
The engine currently tries to load audio resources as Web Audio by default. If it detects that the browser does not support loading Web Audio, it will roll back to the DOM Audio mode.

```

Reply:
这里在专业术语里边，应该用 fall back 比较准确一点